app/lib/nomis_client/court_cases.rb

```diff
@@ -3,13 +3,21 @@
 module NomisClient
   class CourtCases < NomisClient::Base
     class << self
-      ACTIVE_CASES_FILTER = 'activeOnly=true'
+      def get(booking_id, filter_params)
+        query = filter_query(filter_params)
 
-      def get(booking_id)
-        court_cases_route = "/bookings/#{booking_id}/court-cases?#{ACTIVE_CASES_FILTER}"
+        court_cases_route = "/bookings/#{booking_id}/court-cases#{query}"
 
         NomisClient::Base.get(court_cases_route).body
       end
+
+    private
+
+      def filter_query(filter_params)
+        return if filter_params.blank?
+
+        "?activeOnly=#{filter_params.fetch(:active, 'true')}"
```

Comment:
it's good to have :active, 'true' as default value, 
anyway this behaviour is kind of hidden, it could be explicit if you put it as default params for this method.

Reply:
We have both so if we pass filter params that don't include the active key this code will force it's default to be true.